backend/src/nodes/properties/inputs/generic_inputs.py

```diff
@@ -166,3 +167,26 @@ def VideoTypeDropdown() -> DropDownInput:
             },
         ],
     )
+
+
+def BlendModeDropdown() -> DropDownInput:
+    """Blending Mode option dropdown"""
+    return DropDownInput(
+        "Blend Mode",
+        [
+            {"option": "Normal", "value": bm.NORMAL},
+            {"option": "Multiply", "value": bm.MULTIPLY},
+            {"option": "Darken", "value": bm.DARKEN},
+            {"option": "Lighten", "value": bm.LIGHTEN},
+            {"option": "Add", "value": bm.ADD},
```

Comment:
Add and multiply should be grouped together IMO.

```suggestion
            {"option": "Add", "value": bm.ADD},
            {"option": "Multiply", "value": bm.MULTIPLY},
            {"option": "Darken", "value": bm.DARKEN},
            {"option": "Lighten", "value": bm.LIGHTEN},
```

Reply:
Oh whoops. I actually meant to sort these after I got them added and just forgot.

Reply:
Sorted based on categories a la Photoshop and GIMP: Normal, Darken, Lighten, Contrast, Inversion, Cancelation. (I'm not 100% sure exactly what Reflect/Glow qualify as, but they seem to fit in Contrast by the way they behave.)